docs/wiki/basic-tutorials/embeds.md

```diff
@@ -39,6 +39,34 @@ EmbedBuilder embed = new EmbedBuilder()
 // Send the embed
 channel.sendMessage(embed);
 ```
+All this is a bit confusing :thinking: so let's break it down!
+
+## :bone: Basics
+To start using the embed builder, you need to instantiate it like any other class and then you can start acessing it's methods. We'll go through the basic ones here:
+
+* `EmbedBuilder.setTitle("")`- Most of the time you will want a title for your embed, won't you :question: This sets the title of the embed, the top-most words in the embed.
```

Comment:
EmbedBuilder#setTitle() is like the other methods not static. You shouldn't use a dot there.

Reply:
And I don't think you should use "" as a placeholder for Strings. You can't even set it to an empty String.